source/src/main/java/org/cerberus/core/service/datalib/impl/DataLibService.java

```diff
@@ -772,7 +780,7 @@ private AnswerList<HashMap<String, String>> getDataObjectList(TestDataLib lib, H
                                 Integer sqlTimeout = parameterService.getParameterIntegerByKey("cerberus_propertyexternalsql_timeout", system, 60);
 
                                 //performs a query that returns several rows containing n columns
-                                responseList = sqlService.queryDatabaseNColumns(connectionName, lib.getScript(), rowLimit, sqlTimeout, system, columnList, columnsToHide, execution);
+                                responseList = sqlService.queryDatabaseNColumns(connectionName, lib.getScript(), rowLimit, sqlTimeout, system, columnList, columnsToHide, ignoreNonMatchedSubdata ? ISQLService.IGNORE_NON_MATCHED_COLUMNS : ISQLService.DO_NOT_IGNORE_NON_MATCHED_COLUMNS, defaultSubdataValue, execution);
```

Comment:
Same here.

Reply:
These variables were used for better readability. The latest commit (#17ba4e0) removes them.